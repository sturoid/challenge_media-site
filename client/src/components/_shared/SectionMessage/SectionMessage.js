import React from 'react';
import PropTypes from 'prop-types';
import container from './SectionMessage.styles';

const SectionMessage = ({ text }) => <div className={container}>{text}</div>;

SectionMessage.propTypes = {
  text: PropTypes.string.isRequired
};

export default SectionMessage;
