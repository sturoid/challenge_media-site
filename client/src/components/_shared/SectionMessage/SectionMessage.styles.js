import { css } from 'emotion';
import { greyLight } from '../../../lib/colors.styles';

const container = css`
  color: ${greyLight};
  font-size: 24px;
  margin-top: 20px;
  text-align: center;
`;

export default container;
