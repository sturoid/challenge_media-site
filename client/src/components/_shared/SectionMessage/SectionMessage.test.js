import React from 'react';
import renderer from 'react-test-renderer';
import SectionMessage from './SectionMessage';

describe('<SectionMessage />', () => {
  it('renders correctly and matches snapshot', () => {
    const tree = renderer
      .create(<SectionMessage text="This is a message" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
