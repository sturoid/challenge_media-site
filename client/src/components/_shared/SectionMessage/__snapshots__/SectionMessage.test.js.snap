// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionMessage /> renders correctly and matches snapshot 1`] = `
<div
  className="css-arfcei"
>
  This is a message
</div>
`;
